build.cmd

```diff
@@ -431,18 +429,15 @@ if defined __MscorlibOnly (
     exit /b 0
 )
 
-REM Consider doing crossgen build of mscorlib unless we are skipping it intentionally
-if /i "%__BuildArch%" == "arm64" (
-    if not defined __DoCrossgen (
-        echo %__MsgPrefix%Skipping Crossgen
-        goto SkipCrossGenBuild
-    )
-)
-
 echo %__MsgPrefix%Generating native image of mscorlib for %__BuildOS%.%__BuildArch%.%__BuildType%
 
 set "__CrossGenMScorlibLog=%__LogsDir%\CrossgenMScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
-"%__BinDir%\crossgen.exe" "%__BinDir%\mscorlib.dll" > "%__CrossGenMScorlibLog%" 2>&1
+set "__CrossgenExe=%__BinDir%\crossgen.exe"
+if /i "%__BuildArch%" == "arm64" (
```

Comment:
Instead of this "if", I would suggest adding a location for crosstarget component binaries and pick it up from there.